tests/migrations/system/test_sample_migrations.py

```diff
@@ -0,0 +1,299 @@
+from typing import List, Optional
+
+from datastore.migrations import (
+    BaseEvent,
+    BaseMigration,
+    CreateEvent,
+    DeleteFieldsEvent,
+    ListUpdateEvent,
+    MigrationKeyframeAccessor,
+    PositionData,
+    RestoreEvent,
+    UpdateEvent,
+)
+from datastore.shared.util import (
+    collection_and_id_from_fqid,
+    fqid_from_collection_and_id,
+)
+
+
+class RenameField(BaseMigration):
```

Comment:
I find it a bit inconsistent that this class is global while the other ones are nested. I would suggest to move them to a separate file for easier overview and access. Maybe this class should even live in the main code so a migration can directly inherit from this class and only modify the field names?

Reply:
Hm, I want to keep this inside this migration file since `RenameField` is currently not generic. It is separate since I use it often. I'm not sure if moving it is not over-complicating it.

If you need this later when writing real migrations, you can refactor it ;)

Reply:
ok